gtest/src/system.rs

```diff
@@ -111,4 +111,22 @@ impl System {
             .or_insert_with(Vec::default);
         Mailbox::new(program_id, &self.0)
     }
+
+    /// Add value to the actor.
+    pub fn add_value_to<ID: Into<ProgramIdWrapper>>(&self, id: ID, value: Balance) {
+        let actor_id = id.into().0;
+        self.0.borrow_mut().add_value_to(&actor_id, value);
+    }
+
+    /// Return actor balance (value) if exists.
+    pub fn actor_balance<ID: Into<ProgramIdWrapper>>(&self, id: ID) -> Option<Balance> {
+        let actor_id = id.into().0;
+        self.0.borrow().actor_balance(&actor_id)
+    }
+
+    /// Claim the user's value from the mailbox.
+    pub fn claim_value_from_mailbox<ID: Into<ProgramIdWrapper>>(&self, id: ID) {
```

Comment:
this method should also be provided for `Mailbox` API

Reply:
Added `Mailbox::claim_value` function (https://github.com/gear-tech/gear/pull/1125/commits/7a91463fbb8dfa1c726d1ac88601641d948498d5).